core/src/main/java/com/linecorp/armeria/server/VirtualHost.java

```diff
@@ -84,6 +85,9 @@
     private final boolean shutdownAccessLogWriterOnStop;
     private final ScheduledExecutorService blockingTaskExecutor;
     private final boolean shutdownBlockingTaskExecutorOnStop;
+    @Nullable
+    private final EventLoopGroup serviceWorkerGroup;
```

Comment:
I know it's confusing, but we have to remove `@Nullable` here, like we did for `defaultServiceNaming`, because a user will never see `null`.
```suggestion
    private final EventLoopGroup serviceWorkerGroup;
```